_pages/software-tools/virtual-desktop.md

```diff
@@ -0,0 +1,56 @@
+---
+title: Citrix
+---
+
+Citrix is a Windows virtual desktop client that can be used to connect to GSA's intranet, or test something in Internet Explorer.
+
+## Setup
+
+In order to launch Citrix, you will need to have the Citrix Client installed on your device. If you are connecting from a GSA Laptop you will already have the Citrix Client installed.
+
+If this is a non-GSA Device you should install the Citrix Workspace App before moving to the next step. The Citrix Workspace App can be downloaded from Citrix at the below links for Windows or Apple Mac respectively:
+
+[Citrix Workspace app for Windows](https://www.citrix.com/downloads/workspace-app/windows/workspace-app-for-windows-latest.html)
+
+[Citrix Workspace app for Mac](https://www.citrix.com/downloads/workspace-app/mac/workspace-app-for-mac-latest.html)
+
+## Logging into the Citrix VDI
+
+These instructions are pulled from the [InSite page](https://insite.gsa.gov/employee-resources/information-technology/do-it-yourself-self-help/telework-technology/citrix/citrix-and-citrix-workspace):
```

Comment:
I generally don't want to duplicate instructions since:

- It adds documentation for us to maintain, when there is already someone else doing so
- It gets confusing if they drift; people don't know which to follow

but we can revisit later.